src/cljc/matcher_combinators/core.cljc

```diff
@@ -143,6 +144,12 @@
      ::result/weight 1})
   (-name [_] 'absent))
 
+(defmethod clojure.pprint/simple-dispatch Absent [absent]
+  (pr (-name absent)))
```

Comment:
this should technically go in the `printer` namespace but it would make things a bit more messy in terms of dependencies / using `-name` outside of `core` (which I guess is fine, so I'm okay with pushback to move it there)

anyways before this
```
FAIL in (mismatcher-matcher) (matchers_test.clj:384)
assert presence of key via double negation
expected: (match? (m/mismatch {:a m/absent}) {:b 1})
  actual: (mismatch
 expected mismatch from:
 {:a {}}
 {:b 1})

```

and after

```
FAIL in (mismatcher-matcher) (matchers_test.clj:384)
assert presence of key via double negation
expected: (match? (m/mismatch {:a m/absent}) {:b 1})
  actual: (mismatch
 expected mismatch from:
 {:a absent}
 {:b 1})
```